docs/get-started/advisors-details.md

```diff
@@ -0,0 +1,172 @@
+| Generic Performance| mongodb\_multiple\_services | Warns if multiple mongod services are are detected running on a single node. | MongoDB - Multiple mongod Services |
+| Replication Performance| mongodb\_chunk\_imbalance | Warns if the distribution of chunks across shards is imbalanced.| MongoDB Sharding - Chunk Imbalance Across Shards |
+| Replication Performance| mongodb\_oplog\_size_recommendation |Warns if the oplog window is below a 24-hour period and provides a recommended oplog size based on your instance. | MongoDB - Oplog Recovery Window is Low |
+| Replication Performance| mongodb\_replication\_lag | Warns if the replica set member lags behind the primary by more than 10 seconds. | MongoDB Replication Lag |
+| Index Query| mongodb\_shard\_collection\_inconsistent\_indexes | Warns if there are inconsistent indexes across shards for sharded collections. Missing or inconsistent indexes across shards can have a negative impact on performance. | MongoDB Sharding - Inconsistent Indexes Across Shards |
+| Index Query| mongodb\_unused\_index | Warns if there are unused indexes on any database collection in your instance. This requires enabling the "indexStats" collector. | MongoDB - Unused Indexes |
+| Authentication Security| mongodb_auth | Warns if MongoDB authentication is disabled. | MongoDB authentication |
+| Authentication Security| mongodb\_localhost\_auth_bypass | Warns if MongoDB localhost bypass is enabled. | MongoDB localhost authentication bypass enabled |
+| Configuration Security| mongodb\_authmech\_scramsha256 | Warns if MongoDB is not using the default SHA-256 hashing function as its SCRAM authentication method. | MongoDB Security AuthMech Check |
+| Connection Security| mongodb_bindip | Warns if the MongoDB network binding is not set as Recommended. | MonogDB IP Bindings |
+| CVE Security| mongodb\_cve\_version | Shows an error if MongoDB or Percona Server for MongoDB version is older than the latest version containing CVE (Common Vulnerabilities and Exposures) fixes. | MongoDB CVE Version |
+
+
+### MySQL
+| Advisor| Check Name | Description | Summary |
+| :--------- | :---------- | :--- |:--- |
+|Connection Configuration| mysql\_configuration\_max\_connections\_usage | Checks for MySQL max_connections configuration option for maximum utilization | Check max connections usage |
+| Generic Configuration | mysql\_automatic\_sp\_privileges\_enabled | This check reviews the automatic\_sp\_privileges configuration is ON. | Checks if automatic\_sp\_privileges configuration is ON. |
+| Generic Configuration | mysql\_config\_binlog\_retention\_period | Binlogs should not be rotated too often, except very specific cases. | Binlogs retention check |
+| Generic Configuration | mysql\_config\_binlog\_row\_image | Please consider setting binlog\_row\_image=FULL. | Binlogs raw image is not set to FULL |
```

Comment:
The Summary does not match the format of the other ones, which seems to be a Title format. Instead it kind of duplicates the description. Should it be "automatic\_sp\_privileges check"?